soccer/planning/CompositePath.cpp

```diff
@@ -93,11 +93,18 @@ void CompositePath::draw(SystemState* const state, const QColor& color,
 
 float CompositePath::getDuration() const { return duration; }
 
-boost::optional<MotionInstant> CompositePath::destination() const {
+MotionInstant CompositePath::start() const {
     if (paths.empty()) {
-        return boost::none;
```

Comment:
So I'm guess infinite paths aren't a thing anymore? Not that we really every had a real use case for it.